plugins/source/postgresql/client/sync.go

```diff
@@ -118,27 +122,29 @@ func syncTable(ctx context.Context, tx pgx.Tx, table *schema.Table, res chan<- m
 		return err
 	}
 	defer rows.Close()
+
 	for rows.Next() {
 		values, err := rows.Values()
 		if err != nil {
 			return err
 		}
 
-		arrowSchema := table.ToArrowSchema()
-		rb := array.NewRecordBuilder(memory.DefaultAllocator, arrowSchema)
-		for i := range values {
-			val, err := prepareValueForResourceSet(arrowSchema.Field(i).Type, values[i])
```

Comment:
Should we do the same in `cdc.go` and then remove `prepareValueForResourceSet`, rather than having similar logic in two places?

Reply:
9c693b3086fde8d5787fb03f8ad6e480ca06cc65